app-backend/batch/src/main/scala/com/azavea/rf/batch/aoi/UpdateAOIProject.scala

```diff
@@ -56,16 +56,16 @@ case class UpdateAOIProject(projectId: UUID)(implicit val xa: Transactor[IO]) ex
       <html>
         <p>${user.name},</p><br>
         <p>You have ${sceneCount} new scenes updated to your AOI project "${project.name}"! You can access
-        these new scenes <a href="https://${platformHost}/projects/edit/${project.id}/scenes" target="_blank">here</a> or any past
```

Comment:
Why the switch from `https`? Pretty sure we'll want to keep these links as `https` since the application is hosted with SSL